urdf/include/urdf/model.h

```diff
@@ -56,6 +57,7 @@ class Model: public ModelInterface
   bool initFile(const std::string& filename);
   /// \brief Load Model given the name of a parameter on the parameter server
   bool initParam(const std::string& param);
+  bool initParamWithNodeHandle(const std::string& param, const ros::NodeHandle& nh);
```

Comment:
Ah it is different. In `kinetic-devel` calling `initParamWithNodeHandle("parameter")` is identical to `initParam("parameter")`. In this PR calling `initParamWithNodeHandle("parameter")` would fail to compile.

I think this is OK since code written for indigo would still work after moving to kinetic. It's only an issue if someone tries to port kinetic code backwards to indigo.

Having a default value for the `NodeHandle` parameter in kinetic doesn't add value since there is already `initParam`, but it's too late to change. Maybe the default value could be removed in ROS melodic?